util/mini-tt2.pl

```diff
@@ -491,15 +459,15 @@
         }
 
     } elsif (!$passthrough) {
-        if (/^ ( (.*? \W) ([_a-zA-Z]\w*) \( ) .*? [^\s\{;\\] \s* $/x) {
+        if (/^ ( (.*? \W) ([_a-zA-Z]\w*) \( ) .*? [^\s\{;] \s* $/x) {
             # found a function call
             my ($prefix, $indent);
             ($prefix, $indent, $func_name) = ($1, $2, $3);
             if ($prefix !~ /^\w+ /) {
                 my $len = length $prefix;
                 if ($raw_line =~ /^ (.*? \w \( ) /x) {
                     my $raw_len = length $1;
-                    if ($prefix !~ m{^ (?: \# | \s* /\* )}x && !m/\\$/) {
+                    if ($prefix !~ m{^ (?: \# \s*(?!:define) | \s* /\* )}x) {
```

Comment:
Reusing the function handling code for macros can be dangerous. How do you handle interleaved macros and C functions?

Reply:
@spacewander Have you checked the newly generated C source from http lua module and stream lua module? Any regressions observed?

Reply:
@agentzh 
I just submit a bugfix commit in this branch.
I ran the latest version of this branch to generate http lua module and the stream one, no regressions have been seen yet.